### PR TITLE
Improve naming and organization of some manual articles

### PR DESCRIPTION
**Pull request by YuriSizov**
This PR includes several changes aimed at improving sidebar navigation or overall organization of the documentation.

- In "Platforms" HTML5/Web platform pages were moved to their own subfolder, `web`;
- In "3D" Global Illumination pages were moved to their own subfolder, `global_illumination`;
- "User Interface (UI)" lost its excessive capitalization to become "User interface (UI)";
- "Input and Output (I/O)" was renamed to "File and data I/O";
- "Inputs" was renamed to "Input handling";
- "2D" and "3D" sections had their tutorials grouped to reduce the size of the initial list, give some order, and ultimately improve navigation;
- Also some no-longer in-use images were dropped.

After these changes the top level of the "Manual" category looks [like this](https://user-images.githubusercontent.com/11782833/221422869-e3dfe87d-f10f-4eec-ba81-beae063ab0c6.png).
And here are the 2D and 3D sections:

chrome_2023-02-26_17-13-28
chrome_2023-02-26_17-13-43

-----

All pages which changed their path were added to redirects.

### CODE REVIEW

**Review by Calinou — APPROVED** [review]

File: _tools/redirects/redirects.csv [hunk 1]
```diff
@@ -324,7 +324,13 @@ source,destination
 /reference/shading_language.html,/learning/features/shading/shading_language.html
 /reference/unity_to_godot.html,/learning/editor/unity_to_godot.html
 /reference/variant_class.html,/development/cpp/variant_class.html
+introduction_to_global_illumination
+/tutorials/3d/faking_global_illumination.html,/tutorials/3d/faking_global_illumination.html
 /tutorials/3d/inverse_kinematics.html,/community/tutorials/3d/inverse_kinematics.html
+/tutorials/3d/reflection_probes.html,/tutorials/3d/reflection_probes.html
+/tutorials/3d/sdfgi.html,/tutorials/3d/using_sdfgi.html
+/tutorials/3d/using_lightmap_gi.html,/tutorials/3d/using_lightmap_gi.html
+/tutorials/3d/using_voxel_gi.html,/tutorials/3d/using_voxel_gi.html
```
Comment (Calinou) [inline _tools/redirects/redirects.csv:333]: The source and destination URLs seem identical, is that expected?

Comment (YuriSizov) [reply _tools/redirects/redirects.csv:333]: Oops, no, of course not. I'll fix it in a moment.

Comment (YuriSizov) [reply _tools/redirects/redirects.csv]: Done!

**Review by akien-mga — APPROVED** [review]
Looks great! You can rebase the temporary fix away.

**Comment by YuriSizov** [comment]
> You can rebase the temporary fix away.

@akien-mga Wait, but I don't think the CI will pass since the class ref hasn't been synced with this repo yet?

**Comment by YuriSizov** [comment]
I don't remember how we resolved this before, if at all, but naturally updating class ref first also fails the CI for that particular commit 🙃 Should be good after both changes are merged now.

**Pull request merged**